### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by Mehria-Rahmani**
The content of  page are personalized. Choose a topic that is different.

design that i followed:  [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:

Colors.
Typography: font face, size and weight.
Layout: composition and space between elements.
The pages look almost identical to the original design. Small adjustments like text or image changes are acceptable.

build  these 2 pages:
The home page.
The about page.
Each of these pages  have versions for 2 different screen sizes:

Mobile: up to 768px wide.
Desktop: 768px or wider.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,44 @@
...
 
-Feel free to check the [issues page](../../issues/).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
+# Acknowledgments
+To my learning partners and Standup team members
```
Comment (ShoiraTa) [inline README.md:40]: - [ ] Kindly add credits to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: about.html [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+            <h3>2020</h3>
+            <p>AYP Summit in Asia</p>
+          </div>
+        </div>
+      </div>
+      <hr class="full-horizontal-line" />
+    </section>
+    <footer class="footer flex-column-center" id="contact">
+      <h3 class="contact light-black heading-medium">Contact</h3>
+      <hr class="horizontal-line text-center" />
```
Comment (ShoiraTa) [inline about.html:175]: - [ ] in Desktop Size, About Page's footer should have a dark background

Your design  | Provided design
------------- | -------------
Screenshot 2022-06-29 at 17 15 27  | Screenshot 2022-06-29 at 17 14 57
.

File: about.html [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+        Asia
+      </p>
+      <div class="activities">
+        <div class="past-first-activity act">
+          <div class="first-activity-content flex-column-center">
+            <h3>2021</h3>
+            <p>AYP Summit in Asia</p>
+          </div>
+        </div>
+        <div class="past-second-activity act">
```
Comment (ShoiraTa) [inline about.html:164]: - [ ] the background image in the first card is missing, kindly add a background image 👍🏻 
Screenshot 2022-06-29 at 17 17 09

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @Mehria-Rahmani ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @Mehria-Rahmani, 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there


 # Status: Requires Changes ♻️
_Check the comments under the review._

# Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+            <h3>2020</h3>
+            <p>AYP Summit in Asia</p>
+          </div>
+        </div>
+      </div>
+      <hr class="full-horizontal-line" />
+    </section>
+    <footer class="footer flex-column-center" id="contact">
+      <h3 class="contact light-black heading-medium">Contact</h3>
+      <hr class="horizontal-line text-center" />
```
Comment (Gambit142) [inline about.html:176]: - [ ] Great work making the changes requested by the previous reviewers. To make your project complete, kindly implement the final change requested by the previous reviewers by making the `background-color` of the `footer` tag with class `.footer` on the about page `#272a31`. After doing this, you can adjust the `color` of the fonts to be a bright `color` for accessibility purposes. Implementing this change will ensure that your project is identical to that of the template provided by Cindy Shin. Kindly implement this change 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @Mehria-Rahmani 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Added background-image to cards on about page ✔️ 
- Gave credits to Cindy Shin the designer of the template used ✔️ 


 ## Required Changes ♻️
_Kindly Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+      </button>
+      <a class="see-program"><u>SEE THE WHOLE PROGRAM</u></a>
+    </section>
+    <section class="featured-speakers flex-column-center" id="speakers">
+      <h2 class="featured-speakers-header light-black heading-medium">
+        Featured Speakers
+      </h2>
+      <hr class="horizontal-line text-center" />
+      <div class="speakers-list"></div>
+      <button
```
Comment (ShoiraTa) [inline index.html:185]: - [ ] OPTIONAL: The featured speakers section was aligned in two columns when I first reviewed, now it looks like the changes you made affected this section and it became a single column. I strongly recommend to bring back the previous implementation 👍🏻 

Your design  | Behance design
------------- | -------------
Screenshot 2022-06-30 at 08 45 21 | Screenshot 2022-06-30 at 08 47 03

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @Mehria-Rahmani , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mehria-Rahmani** [comment]
Thank u so much

File: index.html (hunk 5 above)
Comment (milen-ale) [inline index.html:185]: - [ ] Great work making the changes requested by the previous reviewers. To make your project complete, kindly implement the final change requested by the previous reviewers The featured speakers section was aligned in two columns when I first reviewed it, now it looks like the changes you made affected this section and it became a single column. I strongly recommend to bring back the previous implementation 👍🏻

**Pull request merged**